default-ht-config.yml

```diff
@@ -0,0 +1,43 @@
+# This is the default config that is being provided along with the collector
+# image.
+
+extensions:
+  health_check:
+  pprof:
+    endpoint: 0.0.0.0:1777
+  zpages:
+    endpoint: 0.0.0.0:55679
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+  opencensus:
+  jaeger:
+    protocols:
+      grpc:
+      thrift_binary:
+      thrift_compact:
+      thrift_http:
+  zipkin:
+
+processors:
+  batch:
+
+exporters:
+  kafka:
+    protocol_version: 2.0.0
+    brokers:
+      - ${BROKER}:9092
```

Comment:
Is there a way to keep the default value?
```
${KAFKA_BROKERS:-kafka-zookeeper}
```
If, so, add even for topic as well
topic: ${KAFKA_TOPIC:-jaeger-spans}

And also take the entire string - ${BROKER:-bootstrap:9092}

Reply:
It doesn't work with assigning default values. sample config [here](https://github.com/open-telemetry/opentelemetry-collector/commit/047b0f38e6816b12ffff10fc276eee032f4ad987) suggest the same. 

So, we need to stick to using static environment variables, I am guessing. 

```
hypertrace-collector    | 2021/03/04 13:05:21 application run finished with error: cannot setup pipelines: cannot build builtExporters: error creating kafka exporter: kafka: client has run out of available brokers to talk to (Is your cluster reachable?)
```

Reply:
Yeah I was thinking the same. I will work this out https://github.com/open-telemetry/opentelemetry-collector/issues/2534.